Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -1480,7 +1480,7 @@ MenuList[1]="Zum Spielen über diesem Server brauchst Du eine neuere Version von
 
```

Comment:
Another suggestion that I can't add myself:

Line 1459: "Leveleinstellung." → maybe "Schwierigkeitslevel" or "Skill-Level-Einstellung"? 

Line 1461: "Kein freundliches Feuer" → "Kein Eigenbeschuss" or "Kein Kameradenschaden" maybe? Unsure here, but the original phrasing sounds a little off. Maybe stick with "Kein Friendly-Fire" to conform to the following HelpMessage?

Line 1465: "Waffen bleiben" → "Waffen bleiben" for consistency reasons.

Line 1471: "Wenn aktiviert, werden Waffen sofort wieder aufgetaucht." → "Wenn aktiviert, spawnen Waffen sofort erneut." or "Wenn aktiviert, werden Waffen direkt neu gespawnt."

Line 1473: "Sofortiges Respawn von Gegenständen" → "Sofortiger Respawn von Gegenständen"? Debatable which article Respawn would have, but I know it as "der Respawn".

Line 1475: "Wenn diese Option aktiviert ist, werden die Elemente sofort wieder angezeigt." → "Wenn aktiviert, werden Items direkt wieder neu gespawnt."

Line 1479: "Zum Spielen über diesem Server brauchst Du eine neuere Version von Unreal. Möchtest Du eine neuere Version von der OldUnreal-Website herunterladen?" → "Zum Spielen auf diesem Server brauchst du eine neuere Version von Unreal. Möchtest du eine neuere Version von der OldUnreal-Website herunterladen?"